src/pages/Tutorials.tsx

```diff
@@ -44,26 +40,46 @@ const cardContainerPadding = () => {
 const VideoContainer = styled('div', (props: ThemeProps) => ({
   className: 'video-container',
   display: 'flex',
```

Comment:
I believe adding 
`allow: 'accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture; allowFullScreen',`
would give the option for fullscreen as well.

Reply:
Scratch that, found the right spot, will be posted in additional commit.